### PR TITLE
Get api

### PR DESCRIPTION
**Pull request by KaedenOC**

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                                <div className="text-center mx-auto">
+                                    <p>{this.state.cityData.display_name}</p>
+                                    <p>Latitude:{this.state.cityData.lat}</p>
+                                    <p>Longitude:{this.state.cityData.lon}</p>
+                                </div>
+                            )}
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col className="d-flex justify-content-center align-items-center">
```
Comment (IkeSteoger) [inline src/components/Main.js:91]: oh wow I didn't even know you could use all these class names!

File: src/components/Main.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                                    <div className="alert alert-danger" role="alert">
+                                        {this.state.errorMessage}
+                                    </div>
+                                </div>
+                            ) : (
+                                <div className="text-center mx-auto">
+                                    <p>{this.state.cityData.display_name}</p>
+                                    <p>Latitude:{this.state.cityData.lat}</p>
+                                    <p>Longitude:{this.state.cityData.lon}</p>
+                                </div>
```
Comment (IkeSteoger) [inline src/components/Main.js:86]: I like this div but you could turn it into a cool Card!!

File: src/components/Main.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
+import { Component } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import axios from 'axios';
+import { Container, Row, Col } from 'react-bootstrap'
```
Comment (K1ng-T0ast) [inline src/components/Main.js:5]: Nice job putting this all in one line! You can also add the Form and Button to the same line ;)

File: src/components/Main.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+            //TODO set state with the error boolean and error message
+            this.setState({
+                error: true,
+                errorMessage: error.message
+            })
+        }
+
+    }
+
+    render() {
```
Comment (K1ng-T0ast) [inline src/components/Main.js:58]: Just pretend this is the first you're reading of this, but try putting your Form, Card, and Alert in separate components!

**Pull request merged**